src/Committee/Voter/CreateCommitteeVoter.php

```diff
@@ -17,33 +17,31 @@ public function __construct(CommitteeManager $manager)
         $this->manager = $manager;
     }
 
-    public function vote(TokenInterface $token, $subject, array $attributes)
+    protected function supports($attribute, $subject): bool
```

Comment:
the method signature needs to be the same as its parent:
`protected function supports($attribute, $subject)`